boilerplate/Gruntfile.coffee

```diff
@@ -7,8 +7,11 @@ module.exports = (grunt) ->
   grunt.initConfig(
     files:
       web3:
-        "node_modules/embark-framework/js/web3.js"
-
+        src: [
+          "node_modules/embark-framework/js/web3.js"
+          "node_modules/embark-framework/js/bluebird.js"
+          "node_modules/embark-framework/js/ether-pudding.js"
+        ]
```

Comment:
Why are these additional files being assigned to "web3" and not placed below under js?